src/main/java/com/github/kokorin/jaffree/process/ProcessHandler.java

```diff
@@ -120,6 +121,24 @@ public synchronized ProcessHandler<T> setArguments(final List<String> arguments)
         return this;
     }
 
+    /**
+     * Overrides the default Executor timeout.
+     * <p>
+     * A null value is interpreted as "use default timeout".
+     *
+     * @param executorTimeoutMillis the new Executor timeout in milliseconds
+     * @return this
+     */
+    public ProcessHandler<T> setExecutorTimeoutMillis(final Integer executorTimeoutMillis) {
```

Comment:
this method should not allow null values. Please, consider using `int` primitive.
I would suggest to check for null value in `FFmpeg.createProcessHandler()` method.